src/DigitalSubscriberAppBanner.tsx

```diff
@@ -335,19 +316,23 @@ export const DigitalSubscriberAppBanner: React.FC<Props> = ({
                 <div css={bottomRightComponent}>
                     <div css={packShot}>
                         <img
-                            src="https://via.placeholder.com/400x300"
+                            src="https://media.guim.co.uk/86d117fc42742412c11918cdce82f47c83f65881/32_113_812_487/812.png"
```

Comment:
Are we happy to hotlink this, rather than including it as part of the component and bundle, directly? I think this is fine, but just checking.

Reply:
Support-frontend have a system for getting images from Grid [here](https://github.com/guardian/support-frontend/blob/920e638c35430cc260acdb1878f37bffa1d12fae/support-frontend/assets/helpers/theGrid.js) - as far as I can tell it ultimately assembles one of these media.guim urls. I did try adding the image into the component as a relative link but it didn't seem to be working in Storybook.

Reply:
One other thing to add. The reader revenue banners specify the image slightly differently:

`https://i.guim.co.uk/img/media/773ead1bd414781052c0983858e6859993870dd3/34_72_1825_1084/1825.png?width=500&quality=85&s=24cb49b459c52c9d25868ca20979defb`

I'm not sure of the difference between i.guim and media.guim, but perhaps the fact that rr are using i.guim is something to learn from and we should do the same? They're also controlling the size/quality via query string args. I _think_ the `s` is a signature for the URL so it can't be tampered with, but I'm not certain how we'd generate that. Asking in the Dotcom channel would probably get some good answers!

Reply:
I've asked on Dotcom, it would be good to know!

Reply:
i.guim contains Fastly's image resizing. So if you don't know your exact crop size you can let that system do it. i.guim is probably more useful for JPEG photos rather than PNG graphics.

Reply:
Excellent. I've got it on i.guim now, shaving off a little bit of file size